src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -121,11 +135,11 @@
             const selectedItemsList = this.container.querySelector('.ibexa-dropdown__selection-info');
 
             if (selected) {
-                const label = element.querySelector('.ibexa-dropdown__item-label').innerHTML;
+                const labelNode = element.querySelector('.ibexa-dropdown__item-label');
+                const label = this.selectedItemLabel ? this.selectedItemLabel : labelNode.innerHTML;
+                const where = selectedItemsList.querySelector('.ibexa-dropdown__selected-item--predefined');
```

Comment:
This `where` variable tells nothing, at least for me. I had to check line 142 to understand its purpose